telemetry/definitions/commonDefinitions.json

```diff
@@ -12,6 +12,24 @@
             "allowedValues": ["CREATE_FAILED", "RUNNING", "DELETED", "DELETE_FAILED", "PAUSED", "OPERATION_IN_PROGRESS"],
             "description": "The current state of the App Runner service"
         },
+        {
+            "name": "awsFiletype",
```

Comment:
I'm being pedantic but 'awsLanguage' might be a better name? More generic too.

Reply:
Many of these are just plain JSON and thus more of a format than language. I was also looking for something that is more distinctive as a topic--I sense that we want something _less_ generic in this case.